.github/workflows/nightly.yaml

```diff
@@ -17,3 +17,48 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           args: --namespace=test-clusters --image=cilium/cilium-test-dev:${{ steps.vars.outputs.sha_short }} "/usr/local/bin/cilium-test-gke.sh" "docker.io/cilium/cilium:latest" "docker.io/cilium/operator-generic:latest" "docker.io/cilium/hubble-relay:latest" "NightlyPolicyStress"
+  baseline-test:
+    name: Start performance test
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Trim git sha
+        id: vars
+        run: echo "::set-output name=sha_short::$(echo ${GITHUB_SHA:0:9})"
+      - uses: actions/checkout@v2
+      - name: Add namespace object
+        run: |
+          cat <<EOT > cilium.yaml
+          apiVersion: v1
+          kind: Namespace
+          metadata:
+            name: cilium-perf
+          ---
+          EOT
+      - name: Generate Cilium manifests
+        run: |
+          helm version
+          helm template cilium ./install/kubernetes/cilium \
+            --namespace cilium-perf \
+            --set global.disableCriticalPriorityClass=true \
+            --set global.nodeinit.enabled=true \
+            --set nodeinit.reconfigureKubelet=true \
+            --set nodeinit.removeCbrBridge=true \
+            --set global.cni.binPath=/home/kubernetes/bin \
+            --set global.gke.enabled=true \
+            --set config.ipam=kubernetes \
+            --set nodeinit.restartPods=true \
+            --set global.cilium.metrics.enabled=true \
+            --set global.prometheus.enabled=true \
+            --set global.operatorPrometheus.enabled=true \
+            --set global.hubble.metrics.enabled="{dns,drop,tcp,flow,port-distribution,icmp,http}" \
+            --set global.prometheus.enabled=true \
+            --set global.operatorPrometheus.enabled=true \
+            --set global.nativeRoutingCIDR=NATIVE_CIDR_PLACEHOLDER \
```

Comment:
I assume that the next step will do some magic and update this PLACEHOLDER variable

Reply:
The magic happens inside the test image itself. We cannot determine `nativeRoutingCIDR` at this point because the cluster is not yet created. Test job will edit Cilium configmap with correct native CIDR before it runs tests.